### ISSUE TITLE
Actually stop using GeoClient

### ISSUE THREAD

**Issue by toolness** [JustFixNYC/who-owns-what#196]
We thought we had ridded ourselves of GeoClient after #132, but as we discovered in #194, we still actually use it for the "export data" CSV endpoint, at the very least.  We should probably get rid of it here too.

**Comment by toolness** [comment]
Ack this is a duplicate of #195, which I just noticed existed. Closing!
